perses/app/setup_relative_calculation.py

```diff
@@ -1023,6 +1034,42 @@ def _resume_run(setup_options):
     else:
         raise("Can't resume")
 
+def _process_overrides(overrides, yaml_options):
+    """
+    Takes in a string of overrides, converts them into a dict, then merges them with the
+    yaml_options dict to override options set in the file
+    """
+
+    overrides_dict = {}
+    for opt in overrides:
+        key, val = opt.split(":")
+
+        # Check for duplicates
+        if key in overrides_dict:
+            raise ValueError(
+                f"There were duplicate override options, result will be ambiguous! Key {key} repeated!"
+            )
+
+        # I don't like this part, but I rather do this then to try and add type checking
+        # and casting in setup_relative.py
+        # We do int then float since slices might need a int, but if we can't make it an
+        # int then it is probably a float, and if we can 't do that, then it is a str.
+
+        # First lets see if we can make it a int:
+        try:
+            val = int(val)
+        except ValueError:
+            # Now try float
+            try:
+                val = float(val)
+            except ValueError:
+                # Just keep it a str
+                pass
```

Comment:
Yeah, you are totally right, this is not pretty. I see a couple of issues here:
* Doing it this way could lead to undesirable behavior, think about the order of things being casted. If I pass a float here to get overwritten it will sucessfully be casted to an int, and I don't think we want to do this. 
* We are then limiting our CLI to be able to change only things that are either int, float or string. There are options that we might want to change that are booleans (such as `h_contraints`, `rmsd_restraint` or `render_atom_map`, to name a few).

To avoid having to deal with the logic ourselves, maybe using something like `pyyaml` (or maybe even `json`?) to actually parse the override string would be better, something like
```python
python_dict = yaml.safe_load(overrides)
```

Reply:
So I agree that using a parser would be better, but then we have to have users either pass in properly formatted yaml/json from the cli OR we are still manipulating their input into yaml/json, we are still parsing it.

Passing in `true` or `false` shouldn't be a problem, I just tested it and things work fine. Let me know what doesn't work for you.

I thought I checked that `int(4.2)` would throw a value error. Ooops!  I pushed a fix though, first I check to see if `int(val)` and `float(val)`are the same object, that should fix the problem with ints getting truncated.

Reply:
> Passing in `true` or `false` shouldn't be a problem, I just tested it and things work fine. Let me know what doesn't work for you.

Ah okay, yes, I just tested this and it seems that the yaml parser takes both `'True'` and `True` as the correct boolean option, so it shouldn't be a problem.